xarray/tests/test_backends.py

```diff
@@ -2485,6 +2490,44 @@ def test_non_existent_store(self) -> None:
         ):
             xr.open_zarr(f"{uuid.uuid4()}")
 
+    @pytest.mark.skipif(
+        not has_zarr_v3, reason="zarr-python <3 did not support async loading"
+    )
+    @pytest.mark.asyncio
+    async def test_load_async(self) -> None:
```

Comment:
If this parallels `test_load` then lets keep this on the base class and use `pytest.skip()` on the netCDF subclasses. That way it's easy to keep the two in sync. Let's add a comment requesting future contributors to keep the two in sync